backend/Carsharing/Hubs/IChatClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Persistence.Chat.ChatEntites.SignalRModels;
+
+namespace Carsharing.ChatHub;
+
+public interface IChatClient
+{
+    Task RecieveMessage(ChatMessage message);
```

Comment:
постфикс Async на async методах
```suggestion
    Task RecieveMessageAsync(ChatMessage message);
```